src/main/webapp/comments.html

```diff
@@ -7,11 +7,18 @@
   </head>
   <body>
     <div>
-      <header><h1>Black Student Insight</h1></header>   
+      <header>
+        <a href="index.html"><h1>Black Student Insight</h1></a>
+      </header>
       <main>
-        <a href="college-info.html">Back to Profile Home</a>  
+        <div class="topnav"> 
+          <a href="college-info.html">School Info</a>
+          <a class="active" href="comments.html">Student Reviews</a> 
```

Comment:
So this navbar will be on every page? Cuz if you click it here it will probably just clear the data from the page

Reply:
Ah, I see. It's supposed to be on both, college-info and comments